From f0e04af91c4b116731f292e3c9025acdcf1b4c59 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 18 Jun 2017 11:36:38 +0100
Subject: [PATCH] daemon: Fix the order of left-side window buttons

This fixes #882 by ensuring that the window decoration style is consistent
between native and CSD window decorations.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/daemon/settings.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/daemon/settings.vala b/src/daemon/settings.vala
index f0c3c2f..d5dd507 100644
--- a/src/daemon/settings.vala
+++ b/src/daemon/settings.vala
@@ -83,7 +83,7 @@ public class SettingsManager
         switch (style) {
         case ButtonPosition.LEFT:
             xset = this.new_filtered_xsetting("close,minimize,maximize:menu");
-            wm_set = "close,maximize,minimize:appmenu";
+            wm_set = "close,minimize,maximize:appmenu";
             break;
         case ButtonPosition.TRADITIONAL:
         default:
-- 
2.13.1

